its/src/test/java/com/sonar/it/csharp/ProjectLevelDuplicationTest.java

```diff
@@ -52,7 +52,7 @@ public void containsOnlyOneProjectLevelIssue() throws Exception {
 
     List<Issues.Issue> issues = getIssues("ProjectLevelDuplication")
       .stream()
-      .filter(x -> x.getRule().startsWith("csharpsquid:"))
+      .filter(x -> x.getRule().startsWith("csharpsquid:S3904"))
```

Comment:
I guess this should be equals, rather than startsWith?

Reply:
Indeed. Changed it to `equals()`.